src/Transloadit.js

```diff
@@ -143,24 +148,27 @@ class TransloaditClient {
       this._lastUsedAssemblyUrl = `${this._endpoint}${urlSuffix}`
 
       // eslint-disable-next-line no-bitwise
-      await pMap(Object.entries(files), async ([, path]) => access(path, fs.F_OK | fs.R_OK), { concurrency: 5 })
+      await pMap(Object.entries(files), async ([, path]) => access(path, fs.F_OK | fs.R_OK), {
+        concurrency: 5,
+      })
 
       // Convert uploads to streams
-      const streamsMap = fromPairs(Object.entries(uploads).map(([label, value]) => {
-        const isReadable = isStream.readable(value)
-        if (!isReadable && isStream(value)) {
-          // https://github.com/transloadit/node-sdk/issues/92
-          throw new Error(`Upload named "${label}" is not a Readable stream`)
-        }
+      const streamsMap = fromPairs(
+        Object.entries(uploads).map(([label, value]) => {
+          const isReadable = isStream.readable(value)
+          if (!isReadable && isStream(value)) {
+            // https://github.com/transloadit/node-sdk/issues/92
+            throw new Error(`Upload named "${label}" is not a Readable stream`)
+          }
 
-        return [
-          label,
-          isStream.readable(value) ? value : intoStream(value),
-        ]
-      }))
+          return [label, isStream.readable(value) ? value : intoStream(value)]
```

Comment:
not sure if this is more readable than the previous 😅

Reply:
Sadly, in some cases the autofixing results in worse code. I had this in the content repo too. It's a downside. All I can say is that after biting the bullet

A) the upsides weigh up to the few cases
B) it's easy to clean these cases up a bit again as you work on the code and get to know what prettier will be doing to it

Reply:
yes, entire files, or: ignore the next expression.